smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/serde/MemberShapeEncodeGeneratable.kt

```diff
@@ -1,32 +0,0 @@
-/*
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0.
- */
-
-package software.amazon.smithy.swift.codegen.integration.serde
-
-import software.amazon.smithy.model.shapes.ShapeType
-
-interface MemberShapeEncodeGeneratable {
```

Comment:
Deleted since there is only one member shape encode implementation.

Also, the categorizations below are no longer used by the member shape encode implementation.